pkg/hubble/parser/seven/parser.go

```diff
@@ -150,6 +151,11 @@ func (p *Parser) Decode(r *accesslog.LogRecord, decoded *flowpb.Flow) error {
 	decoded.TraceContext = p.getTraceContext(r)
 	decoded.Summary = p.getSummary(r, decoded)
 
+	err = correlation.CorrelatePolicy(p.endpointGetter, decoded)
+	if err != nil {
+		p.log.WithError(err).Error("error correlating policy with flow")
+	}
+
```

Comment:
maybe:
```suggestion
	if p.endpointGetter != nil {
		err = correlation.CorrelatePolicy(p.endpointGetter, decoded)
		if err != nil {
			p.log.WithError(err).Error("error correlating policy with flow")
		}
	}
```
to be consistent with pkg/hubble/parser/threefour/parser.go?

also a bit worried about logging an error here. when could `CorrelatePolicy` fail, and what can users do about it?

Reply:
Ah, the function actually doesn't have any cases where it returns any errors, so we could remove it. That being said the function logs errors in cases where it cannot correlate to a policy. For example: if it cannot lookup the endpoint, or if it cannot find a policy.